src/domain/opportunity/opportunity.resolver.fields.ts

```diff
@@ -28,11 +28,7 @@ export class OpportunityResolverFields {
     private opportunityService: OpportunityService
   ) {}
 
-  @Roles(
-    RestrictedGroupNames.CommunityAdmins,
-    RestrictedGroupNames.EcoverseAdmins,
-    RestrictedGroupNames.Members
-  )
+  @Roles(RestrictedGroupNames.Members)
```

Comment:
This will restrict only to the Members, not all levels. Roles need to be added explicitly.